### ISSUE TITLE
Add prompt for `controller.http_port` during install.

### ISSUE THREAD

**Issue by Cooldude2606** [clusterio/clusterio#564]
Tl;dr When running `create` for standalone or controller you should be asked to enter a port number to host the web ui on.

### Motivation
When installing a cluster for the first time it should be as seamless as possible. Currently you run into the situation where the default port is already in use and you can not start the controller. The workaround is to manually edit the json for the controller config, this is simple to do but is not seamless.

### Solution
You should be prompted on installation to enter a port number for the web ui to be hosted on. This way a known available port can be selected allowing the controller to start without any issues. After which the web interface can be used to edit the remaining config values which is both safer and less technical.
